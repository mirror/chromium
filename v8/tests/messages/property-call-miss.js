function A() { this.foo = function () {} }
function call_foo(obj) { return obj.foo(); }
call_foo(new A());
call_foo(null);
