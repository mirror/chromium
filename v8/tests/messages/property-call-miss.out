property-call-miss.js:2: Uncaught TypeError: Cannot call method 'foo' of null
function call_foo(obj) { return obj.foo(); }
                                    ^
